plugins/CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
 project(plugins)
 
+# libusb-1.0
+pkg_check_modules(LIBUSB_1 IMPORTED_TARGET libusb-1.0)
```

Comment:
why is this here?
It doesn't belong to every plugin

Reply:
You are correct, but it is used in at least 2 directories and I did not want to check modules twice.